IssueCreator/IssueCreator/Dialogs/TextInput.cs

```diff
@@ -0,0 +1,32 @@
+﻿using System;
+using System.Windows.Forms;
+
+namespace IssueCreator
+{
+    public partial class TextInput: Form
```

Comment:
Nit - You could just use `Microsoft.VisualBasic.Interaction.InputBox` if you felt like being lazy and didn't mind washing your hands for an extra 20 seconds.  This is fine too.

Reply:
I'll leave it like this for now -- didn't know that existed!! We can consider this for future!